src/styles/index.js

```diff
@@ -396,10 +396,9 @@ const styles = {
     borderWidth: 1,
     borderColor: colours.lightgray,
     paddingHorizontal: 10,
-    fontSize: 12,
+    fontSize: 19,
     height: 40,
-    flexDirection: 'row',
-    flex: 1
```

Comment:
@minaorangina this was the reason why it was why the errors were squashed in the input field: ```flex: 1``` property. After removing this property error appeared under the input.